### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/MIR#44] [Show and tell]
Post your images of MIR here

**Comment by Toa-Sampa** [comment]
![Screenshot 2022-09-30 20 27 54](https://user-images.githubusercontent.com/67250785/193435512-4c5b928f-a628-4435-9027-0e84a30373c1.png)
![Screenshot 2022-09-30 20 31 11](https://user-images.githubusercontent.com/67250785/193435514-a4b4c9b1-f933-45bc-bf14-c23a9f5548f7.png)
![Screenshot 2022-09-30 01 52 47](https://user-images.githubusercontent.com/67250785/193435516-c9cd60ca-ff54-445b-84ad-2658e8c3395b.png)
